kolibri/plugins/device_management/assets/src/views/UserPermissionsPage/index.vue

```diff
@@ -1,35 +1,66 @@
 <template>
 
   <ImmersiveFullScreen v-bind="{ backPageLink, backPageText }">
+    <!-- TODO should I try and use the baked in auth page? Does this have a URL-->
     <AuthMessage v-if="!isSuperuser" authorizedRole="superuser" />
 
     <h1 v-else-if="user === null">{{ $tr('userDoesNotExist') }}</h1>
 
     <template v-else>
-      <div class="section">
+      <div class="section user-info">
         <h1>
           {{ user.full_name }}
           <span v-if="isCurrentUser">
             ({{ $tr('you') }})
           </span>
         </h1>
-        <h3>{{ user.username }}</h3>
+
+        <table>
+          <tr>
+            <th scope="row">
+              {{ $tr('usernameLabel') }}
+            </th>
+            <td> {{ user.username }} </td>
```

Comment:
Minor comment, but in general throughout the codebase we would write

```html
<td>{{ user.username }}</td>
```

rather than

```html
<td> {{ user.username }} </td>
```

In addition to codebase consistency, once in a while the extra whitespace actually has an unexpected style or layout impact.

The extra inline whitespace is a little harder to spot than the super obvious case above e.g.

```html
<th scope="row">
  {{ $tr('usernameLabel') }}
</th>
```

Reply:
What would the ideal fix be for extra inline whitespace?